app/src/lib/stores/helpers/create-tutorial-repository.ts

```diff
@@ -0,0 +1,141 @@
+import * as Path from 'path'
+
+import { Account } from '../../../models/account'
+import { writeFile, pathExists, ensureDir } from 'fs-extra'
+import { API } from '../../api'
+import { APIError } from '../../http'
+import {
+  executionOptionsWithProgress,
+  PushProgressParser,
+} from '../../progress'
+import { envForAuthentication } from '../../git/authentication'
+import { git } from '../../git'
+import { friendlyEndpointName } from '../../friendly-endpoint-name'
+
+const nl = __WIN32__ ? '\r\n' : '\n'
```

Comment:
[:blood-sob:](https://stackoverflow.com/questions/49968130/line-endings-also-known-as-newlines-in-js-strings)